trade-finance-manager-ui/server/routes/login/configs/loginSso.ts

```diff
@@ -1,19 +1,19 @@
 import express from 'express';
 import { LoginController } from '../../../controllers/login/loginSso/login.controller';
-import { getRouter } from '../../../types/get-router';
+import { GetRouter } from '../../../types/get-router';
 import { EntraIdService } from '../../../services/entra-id.service';
 import { EntraIdConfig } from '../../../configs/entra-id.config';
 import { EntraIdApi } from '../../../third-party-apis/entra-id.api';
 
-export const getLoginSsoRouter: getRouter = () => {
+export const getLoginSsoRouter: GetRouter = () => {
   const entraIdConfig = new EntraIdConfig();
   const entraIdApi = new EntraIdApi({ entraIdConfig });
   const entraIdService = new EntraIdService({ entraIdConfig, entraIdApi });
   const loginController = new LoginController({ entraIdService });
 
   const loginSsoRouter = express.Router();
-  // eslint-disable-next-line @typescript-eslint/no-misused-promises
-  loginSsoRouter.get('/', (res, req) => loginController.getLogin(res, req));
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises, @typescript-eslint/unbound-method
```

Comment:
Looks like we don't need the eslint disabling here - not getting any errors here, although might be my setup

Reply:
I think it's your setup :)